shell/platform/android/image_external_texture_gl.cc

```diff
@@ -86,6 +86,7 @@ void ImageExternalTextureGL::ProcessFrame(PaintContext& context,
 void ImageExternalTextureGL::Detach() {
   image_lru_.Clear();
   gl_entries_.clear();
+  latest_android_image_.Reset();
```

Comment:
If I understand this code right, we should acutally just completely drop the `latest_android_image_` stuff instead of doing this.

If I take this out, the app crashes on resume because it tries to read from a closed image reader.

We will _always_ get memory pressure events on app going to background. We could maybe be more selective about whehter we close the image readers based on the level of memory pressure, but we really should have a better understanding of why Samsung phones are doing this before making a decision on that.